### PR TITLE
Add SAP page for docs

### PR DESCRIPTION
**Pull request by SantiSite**
### Description

Added page for documentation. The path to the page is `/docs/sap`.

### CODE REVIEW

**Comment by SantiSite** [comment]
Hi :hand: @odarbelaeze, where should I put the link to this page?

File: src/components/pages/docs/Sap.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { FC } from "react";
+
+const Sap: FC = () => {
+ return (
+  <article className="prose prose-stone m-auto text-justify max-w-3xl">
+   <h1 className="font-tall uppercase text-leaf">ToS Definitions</h1>
+   <h2 className="uppercase font-tall text-root">Roots</h2>
```
Comment (odarbelaeze) [inline src/components/pages/docs/Sap.tsx:7]: Wrap each of the `h2` and `p` on a section with an ID, add a table of content with links to each section below the `h1`.

**Comment by odarbelaeze** [comment]
Let's remove the title table of contents, and make sure the color of the pinks is blue like the others

**Comment by SantiSite** [comment]
> Let's remove the title table of contents, and make sure the color of the pinks is blue like the others

Of course :+1:

File: src/components/pages/docs/Sap.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { FC } from "react";
+import TableOfContent from "../../common/TableOfContent";
+import { HeadingData } from "../../common/TableOfContent/constants";
+
+const Sap: FC = () => {
+ return (
+  <article className="prose prose-stone m-auto text-justify max-w-3xl">
```
Comment (odarbelaeze) [inline src/components/pages/docs/Sap.tsx:7]: Uhh let's remove `text-justify`.

**Review by odarbelaeze — APPROVED** [review]

**Pull request merged**